src/features/navigator/NarrativeControl/Share.tsx

```diff
@@ -223,44 +341,74 @@ export const Share: FC<{
       }
     }
   }, [dispatch, wsPermsQuery.currentData, wsPermsQuery.isSuccess]);
-  const shareSelectId = useId();
+
+  /* derived values */
   const narrativeIsPublic = usernameShares['*'] === 'r';
   const userPermission: UserPermission =
     (usernameAuthed && usernameShares[usernameAuthed]) || 'n';
-  const userIsAdmin = usernameAuthed && userPermission === 'a';
+  const userIsAdmin = Boolean(usernameAuthed) && userPermission === 'a';
```

Comment:
not sure Boolean is needed, but no need to remove it

Reply:
The `usernameAuthed` is actually a string. I think the name is a bit confusing, so I can understand this comment. Then `userIsAdmin` is passed as a prop to `PublicToggle` which specifies a boolean.